README.md

```diff
@@ -17,15 +17,15 @@ class Comment < ActiveRecord::Base
   belongs_to :post
 
   filterable_by :post_id, :user_id
-  filterable_by :post_author_id do |scope, value|
-    scope.joins(:posts).where(:'posts.author_id' => value)
+  filterable_by :post_author_id do |value|
```

Comment:
so the big change is that you don't now need `scope` as a block argument, but the old method is still supported (although deprecated)

Reply:
value cannot be nil, these blocks are only run `unless value.bank?`